src/helper/util.ts

```diff
@@ -75,7 +75,17 @@ export class ZkWasmUtil {
     return bytes;
   }
 
-  static validateHex(value: string) {
```

Comment:
keep old validateHex as client may had use it and remove the odd checking in it.
Can add this validateI64HexInput which just call this validateHex